index.html

```diff
@@ -66,13 +66,45 @@ <h1>The OpenTF Manifesto</h1>
         </p>
 
         <p>
-          This is why the building blocks of the modern Internet, such as Linux and Kubernetes, are handed over
-          to impartial bodies (the Linux Foundation and the Cloud Native Computing Foundation, respectively) who
-          can ensure that those building blocks can form solid and predictable underpinnings for our industry.
+          We believe that the essential building blocks of the modern Internet, such as Linux, Kubernetes, 
+          and Terraform need to be truly open source: that is the only way to ensure
+          that we are building our industry on top of solid and predictable underpinnings.
         </p>
 
         <p class="font-bold">
-          Our goal: ensure Terraform is handed over to an open source foundation.
+          Our goal: ensure Terraform remains truly open source—always.
+        </p>
+
+        <p>
+          Our aim with this manifesto is to return Terraform to a fully open source license. BSL 
+          is <em>not</em> open source, so this would mean moving Terraform back to the MPL license, or 
+          some other well-known, widely accepted open source license (e.g., APL 2.0). Moreover, we want
+          to be confident that Terraform will remain open source, always, so you don't have to worry
+          about another sudden license change putting everything at risk. 
+        </p>
+
+        <p class="font-bold">
+          Our request of HashiCorp: switch Terraform back to an open source license.
+        </p>
+
+        <p>
+          We ask HashiCorp to do the right thing by the community: instead of going forward with the
+          BSL license change, switch Terraform back to a truly open source license, and commit to keeping
+          it that way forever going forward. That way, instead of fracturing the community, we end up with 
+          a single, impartial, reliable home for Terraform where the whole community can unite to keep 
+          building this amazing ecosystem.
+        </p>
+
+        <p class="font-bold">
+          Our fallback plan: fork Terraform into a foundation.
```

Comment:
I believe this part should be removed.  I think the messaging should be:

1. HCP should move Terraform to an open source license.
2. If Terraform is given to a foundation, we have pledged support for the project.

I think this is positive and no threats.  I feel the threats play into HCP's messaging of people taking advantage.

Reply:
Interesting idea! A few questions:

1. Is the implication that if HashiCorp makes it open source, but not in a foundation, we don't pledge any support?
2. I'll raise this up more broadly shortly, but does (1) imply that the signatories are _only_ signing on for the foundation piece... But not just making it open source? I suspect that's not the message we want to send. We want every company possible to sign this and show support for it...
3. And if HashiCorp does neither of these things... What then? In other words, if it's just a request, it sounds more like a _petition_ than a _manifesto_... Maybe that's OK, but I want to check.